source/includes/important-delayed-replica-set-members.rst

```diff
@@ -0,0 +1,27 @@
+.. important::
+
+   If your replica set contains :doc:`delayed members
+   </core/replica-set-delayed-member>` ensure that the delayed
+   members are hidden and non-voting. 
+
+   Hiding delayed replica set members prevents applications from seeing 
+   and querying delayed data without a direct connection to that member.
+   Making delayed replica set members non-voting means they will not 
+   count towards acknowledging write operations with read concern 
+   :readconcern:`"majority"`.
```

Comment:
Should this be "write concern majority" ? Instead of read concern majority?

Reply:
Updated to write concern.